tests/Paket.Tests/UpdateProcessSpecs.fs

```diff
@@ -736,11 +720,11 @@ let ``SelectiveUpdate updates package that conflicts with transitive dependency
     nuget Ninject.Extensions.Interception
     nuget Ninject.Extensions.Logging""")
     
-    let packageName = PackageName "Ninject.Extensions.Logging"
+    let packageFilter = PackageFilter "Ninject\\.Extensions\\.Logging"
```

Comment:
it would be cool if we would have a function which transform PackageNames into PackageFilter